src/main/docs/guide/toc.yml

```diff
@@ -2,38 +2,22 @@ gettingStarted:
   title: Getting Started
   requirements: What you will need
   howto: How to complete the guide
+googleCloud:
+  title: Create a Google Cloud Project
+  googleActionsApi: Enable Google Actions API 
 writingTheApp:
   title: Writing the Application
-  grailsResponseHandler: Handler that allows Grails to handle requests from Google
-  assistantActionController: Controller for inbound requests from Google to respond to a action
-  mainRequestHandlerFactory: Plumbing to handle the MainRequestHandler
-  mainRequestHandler: Example main handler that says an initial welcome message (goes with textResponseHandler)
-  textRequestHandlerFactory: Plumbing to handle the TextRequestHandler
-  textRequestHandler: Sample handler that echos back something you told it
-  myPermissionRequestHandlerFactory: Plumbing to handle the MyPermissionRequestHandler
-  myPermissionRequestHandler: Sample of asking for a permission
-  colorRequestHandlerFactory: Plumbing to handle the ColorRequestHandler
-  colorRequestHandler: This handles when a user asks for a brighter color for a given color
-  darkerColorRequesteHandlerFactory: Plumbing to handle the DarkerColorRequestHandler
-  darkerColorRequestHandler: This handles when a user asks for a darker color for a given color
-  colorUtils: Small library to help translate between java.awt.Color to give colors names
+  grailsResponseHandler: Handle Request from Google in Grails
+  colorRequestHandlerFactory: Color Handler
   action: action.json is the metadata to tell Google how to handle processing the action calls
-googleActionsApi:
```

Comment:
I disagree with removing these. I think they just need to get updated. I will put these back as I have everything working under the new schema. I think it would be a mistake and gloss over these important details to make it work.